eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -348,7 +347,9 @@ jobs:
 
         CopyWithRelativeFolders "artifacts/"                 $targetFolder "*.binlog"
         CopyWithRelativeFolders "artifacts/"                 $targetFolder "*.log"
-        CopyWithRelativeFolders "artifacts/prebuilt-report/" $targetFolder
+        if ("${{ parameters.buildSourceOnly }}" -eq $True) {
```

Comment:
Is this even necessary at all on Windows?  Would we ever support a source-only windows build?